internal/stack/logs.go

```diff
@@ -22,7 +22,7 @@ func dockerComposeLogsSince(ctx context.Context, serviceName string, profile *pr
 		return nil, fmt.Errorf("can't read application configuration: %w", err)
 	}
 
-	snapshotFile := profile.Path(ProfileStackPath, SnapshotFile)
+	snapshotFile := profile.Path(ProfileStackPath, ComposeFile)
```

Comment:
Nit. I'd rename this variable too, as `composeFile` for instance:

```suggestion
	composeFile := profile.Path(ProfileStackPath, ComposeFile)
```